backend/packages/Upgrade/src/api/models/MonitoredDecisionPointLog.ts

```diff
@@ -7,6 +7,16 @@ export class MonitoredDecisionPointLog extends BaseModel {
   @PrimaryGeneratedColumn()
   public id: string;
 
+  @Column({
+    nullable: true,
+  })
+  public condition: string;
```

Comment:
Change to public condition: string | null; as nullable is true

Reply:
Code updated